src/latex2sympy/latex2antlrJson.cpp

```diff
@@ -1,29 +1,29 @@
-#include <chrono>
 #include <sstream>
 #include "antlr4-runtime.h"
 #include <json/json.h>
 #include <pybind11/pybind11.h>
 #include "LATEXLexer.h"
 #include "LATEXParser.h"
 
+using namespace std;
```

Comment:
So I did a google earlier that says this isn't great practice to import the whole standard namespace, but I don't think it's a big deal. It would be different if this was a huge c++ library, but since it's just one file, I think it's fine.

Reply:
meh I can just revert that commit, no biggie, rather be on the safer side, though it didn't seem to make any difference to the compiled lib

Reply:
ha had a slight bug in the compile.sh